content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+===========
+PDF reports
+===========
+
+Odoo uses HTML and CSS technologies to create reports (PDF documents). HTML elements provide the
+basic structure, while CSS interacts with the elements to establish how they are shown to the user.
+Odoo also uses Bootstrap, a CSS framework, for the grid layout.
+
+.. _studio/pdf-reports/default-layout:
+
+Default layout
+==============
+
+The default layout of reports is managed outside Studio. Go to :menuselection:`Settings -->
+Companies: Document Layout --> Configure Document Layout`. Layout settings apply to all reports but
+only to the current company.
+
+.. tip::
+   Use :guilabel:`Download PDF Preview` to view how the different settings affect the layout of a
+   sample invoice.
+
+.. _studio/pdf-reports/default-layout/layout:
+
+Layout
+------
+
+Four layouts are available.
+
+.. tabs::
+
+   .. tab:: Light
+
+      .. image:: pdf_reports/light.png
+         :align: center
+         :alt: Light report layout sample
+
+   .. tab:: Boxed
+
+      .. image:: pdf_reports/boxed.png
+         :align: center
+         :alt: Boxed report layout sample
+
+   .. tab:: Bold
+
+      .. image:: pdf_reports/bold.png
+         :align: center
+         :alt: Bold report layout sample
+
+   .. tab:: Striped
+
+      .. image:: pdf_reports/striped.png
+         :align: center
+         :alt: Striped report layout sample
+
+.. _studio/pdf-reports/default-layout/font:
+
+Font
+----
+
+Seven fonts are available. Click on the links below to preview them on Google Fonts.
+
+- `Lato <https://fonts.google.com/specimen/Lato#type-tester>`_
+- `Roboto <https://fonts.google.com/specimen/Roboto#type-tester>`_
+- `Open Sans <https://fonts.google.com/specimen/Open+Sans#type-tester>`_
+- `Montserrat <https://fonts.google.com/specimen/Montserrat#type-tester>`_
+- `Oswald <https://fonts.google.com/specimen/Oswald#type-tester>`_
+- `Raleway <https://fonts.google.com/specimen/Raleway#type-tester>`_
+- `Tajawal <https://fonts.google.com/specimen/Tajawal#type-tester>`_
+
+  .. tip::
```

Comment:
that is indeed very useful information, but not sure it should be in the studio section :thinking: (i leave you to decide if there's a better/more wide-reaching place to put it)

Reply:
You are right, it would better fit somewhere else. However, as it is, I would have to create a new page under the Miscellaneous section - a section we should get rid of, someday. And there wouldn't be enough content (or not important enough) to warrant a specific sub-section there I believe.
@jcs-odoo any opinion?